<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginSetForm</class>
 <widget class="QWidget" name="LoginSetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>82</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器名称：</string>
   </property>
  </widget>
  <widget class="LineEditEx" name="lineEditSerName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>76</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>121</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数据库名：</string>
   </property>
  </widget>
  <widget class="LineEditEx" name="lineEditDbName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>116</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="LineEditEx" name="lineEditLoginPwd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="LineEditEx" name="lineEditLoginName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>157</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>162</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>登录名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>205</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="PushButtonEx" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="PushButtonEx" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonEye">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>205</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditEx</class>
   <extends>QLineEdit</extends>
   <header>LineEditEx.h</header>
  </customwidget>
  <customwidget>
   <class>PushButtonEx</class>
   <extends>QPushButton</extends>
   <header>PushButtonEx.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditSerName</tabstop>
  <tabstop>lineEditDbName</tabstop>
  <tabstop>lineEditLoginName</tabstop>
  <tabstop>lineEditLoginPwd</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonTitle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
